### PR TITLE
ARXIVNG-1964 API Terms of Use

### PR DESCRIPTION
**Pull request by erickpeirson**
This adds the Terms of Use for arXiv APIs. See it live: https://marxdown-task-arxivng-1964.development.arxiv.org/help/api/tou

I added links in several relevant places on other help pages.

We still need a link to the code of conduct.

### CODE REVIEW

File: help/api/tou.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  copyright protections. The redistribution of e-prints requires permission
+  from the copyright holder, which may be the author(s) or publisher. arXiv is
+  not the copyright holder on any of the e-prints available through the API. In
+  some cases, submitters have provided permission in advance by submitting
+  their e-print under a permissive Creative Commons license. The vast majority
+  of e-prints are submitted under the [arXiv.org non-exclusive right to
+  distribute](https://arxiv.org/licenses/nonexclusive-distrib/1.0/license.html).
+  For more information about licenses, see https://arxiv.org/help/license.
+- You are free to use descriptive [metadata](https://arxiv.org/help/prep)<sup>1</sup> about arXiv e-prints for
+  non-commercial purposes, under the terms of the [Creative Commons Universal
```
Comment (jimentwood) [inline help/api/tou.md]: I thought all metadata could be used for commercial and non-commercial purposes. Is this a change?

Comment (erickpeirson) [reply help/api/tou.md]: Looks like I overlooked changing the first clause after we added the CC0 language. Will correct.

Comment (erickpeirson) [reply help/api/tou.md]: Fixed in 2bc5edd

**Comment by jimentwood** [comment]
CoC will hopefully be ready next week. You may want to go ahead without it for now if you want to wrap up the API tou.

**Comment by erickpeirson** [comment]
> CoC will hopefully be ready next week. You may want to go ahead without it for now if you want to wrap up the API tou.

Not urgent; this PR can idle.

File: help/api/tou.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  (CC0 1.0) Public Domain
+  Declaration](https://creativecommons.org/publicdomain/zero/1.0/). Metadata
+  are provided by authors, volunteer moderators, arXiv staff, and external
+  partners.
+- You agree to abide by the arXiv Code of Conduct [add link] in all of your
+  interactions with arXiv staff and those involved in its operation, and
```
Comment (jimentwood) [inline help/api/tou.md]: In develop:
/help/policies/code_of_conduct

Comment (erickpeirson) [reply help/api/tou.md]: Added in ae4df54

**Comment by erickpeirson** [comment]
Ok, pushing forward. Proposals for revision can go in new GitHub issues (as appropriate) or via email (if sensitive).

**Pull request merged**